test/format/verify.c

```diff
@@ -172,8 +172,20 @@ table_verify_mirror(WT_CONNECTION *conn, TABLE *base, TABLE *table, const char *
      */
     for (;;) {
         wt_wrap_open_cursor(session, base->uri, checkpoint == NULL ? NULL : buf, &base_cursor);
-        base_id = base_cursor->checkpoint_id(base_cursor);
         wt_wrap_open_cursor(session, table->uri, checkpoint == NULL ? NULL : buf, &table_cursor);
+
```

Comment:
This change took me a couple reads to understand. Please add a comment.

Reply:
Sure, added in ddb0d3a2f05cce1cdde0cdf16fd2bada184e2640